test_infra/src/lib.rs

```diff
@@ -886,7 +980,11 @@ impl Guest {
         Ok(())
     }
 
-    pub fn check_numa_node_distances(&self, node_id: usize, distances: &str) -> Result<bool, Error> {
+    pub fn check_numa_node_distances(
+        &self,
+        node_id: usize,
+        distances: &str,
+    ) -> Result<bool, Error> {
```

Comment:
Good catch. This reminds me that, `cargo fmt` does not work on `test_infra` (likely because we have `exclude = ["test_infra"]` from our top-level toml file). We had to manually do `rustfmt test_infra/src/lib.rs`.

Reply:
I found the quote from Rob's previous commit (https://github.com/cloud-hypervisor/cloud-hypervisor/commit/88c30764e06a9be74220ce1e040331c9827262e1):

```
It must be specified as excluded from the workspace as it must not be
built on non-test targets due to issues with the ssh2 dependency and the
musl toolchain.
```

I also tried to remove the exclude, but ended with the same conclusion that the cross-compilation of `ssh2` (for `musl` and `aarch64`) is still failing and there is no simple solution for it. I guess we will need to keep it for now. :<